Data/ByteString/Builder/Internal.hs

```diff
@@ -204,33 +204,33 @@ trimmedChunkFromBuffer (AllocationStrategy _ _ trim) buf k
 data ChunkIOStream a =
        Finished Buffer a
        -- ^ The partially filled last buffer together with the result.
-     | Yield1 S.ByteString (IO (ChunkIOStream a))
-       -- ^ Yield a /non-empty/ strict 'S.ByteString'.
+     | Yield1 S.StrictByteString (IO (ChunkIOStream a))
+       -- ^ Yield a /non-empty/ 'S.StrictByteString'.
 
 -- | A smart constructor for yielding one chunk that ignores the chunk if
 -- it is empty.
 {-# INLINE yield1 #-}
-yield1 :: S.ByteString -> IO (ChunkIOStream a) -> IO (ChunkIOStream a)
+yield1 :: S.StrictByteString -> IO (ChunkIOStream a) -> IO (ChunkIOStream a)
 yield1 bs cios | S.null bs = cios
                | otherwise = return $ Yield1 bs cios
 
--- | Convert a @'ChunkIOStream' ()@ to a lazy 'L.ByteString' using
+-- | Convert a @'ChunkIOStream' ()@ to a L.LazyByteString using
```

Comment:
The link quotes '' got lost here? This should probably be either `'L.LazyByteString'` or `@LazyByteString@`--whichever you prefer.